app/main.py

```diff
@@ -14,4 +14,13 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    phrase = phrase.lower()
+    letter = letter.lower()
+
+    counter = 0
+    for char in phrase:
+        if char == letter:
+            counter += 1
+
+    return counter
```

Comment:
not bad solution but try to make it in one line. You can use an already existing method that counts the appearance of a char in a string, for example :)